src/sabre_swap/mod.rs

```diff
@@ -318,8 +318,9 @@ pub fn build_swap_map(
             run_in_parallel,
         );
         num_search_steps += 1;
-        if num_search_steps % DECAY_RESET_INTERVAL == 0 {
+        if num_search_steps >= DECAY_RESET_INTERVAL {
             qubits_decay.fill_with(|| 1.);
+            num_search_steps = 0
```

Comment:
Oops, I'm surprised nothing is complaining about this. I guess it's because this statement returns nothing and neither does the else block so the times are aligned. But it feels like I should have the semicolon here

```suggestion
            num_search_steps = 0;
```